src/middlewares/task.middleware.js

```diff
@@ -21,6 +23,8 @@ import { isWin, PACKAGE_MANAGER_CMD } from '../services/platform.service';
 
 import type { Task, ProjectType } from '../types';
 
+const chalk = new chalkRaw.constructor({ level: 3 });
```

Comment:
You should be able to set the level directly on the originally imported object, since we’d expect any uses of it to produce a string with colors.

Reply:
> You should be able to set the level directly on the originally imported object, since we’d expect any uses of it to produce a string with colors.

How do you mean? 

I tried this, didn't work:

```
import chalk from 'chalk';
chalk.level = 3;
```

Plus I'd feel a bit weird about that; while I recognize that in this case it's fine to mutate an import, since we know we'll never want any other value, it still feels like a bad pattern that might be copied.

Reply:
The chalk instance seems to be working great, is there a reason to do it on the original import?

Reply:
Not particularly @melanieseltzer. Let’s keep it as-is.